### PR TITLE
Multi Claim 

### LINKED ISSUE #1816: Claim Multiple Withdrawn Assets via Connection Modal - Design Specification
Currently, we are only able to claim withdrawn xDai on Ethereum Mainnet. With the OMN token launch, we want to be able to withdraw both assets at once, xDAI and xOMN. Therefor, we need to change the visualisation of claimable assets on the connection modal to be prepared to have multiple assets claimable:

Connection

Notes:
- Replace the old visualisation of claimable assets with a collapsed summary component where people can claim all assets through the claim button.
- if you click on the dropdown icon, user is able to see all the claimable asset description and balance
- you can collapse it back into summary component

Figma:
[Design Specification](https://www.figma.com/file/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=2993%3A72065)
[Playground](https://www.figma.com/proto/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=2993%3A72065&viewport=-1114%2C473%2C0.20302648842334747&scaling=min-zoom&page-id=2993%3A72064)

### PR DESCRIPTION
**Pull request by Mi-Lan**
closes #1816

### CODE REVIEW

**Comment by Mi-Lan** [comment]
@hexyls Hey men since you are expert for multi claims I think its best if you take a look at code first to find if i made some mistakes before we spend money on testing...

**Review by hexyls — APPROVED** [review]
Hey man this is slick! Works really well and it looks fantastic. I removed `waitForBridgeMessageStatus` for withdrawals, only needed for deposits 👍 and added a `readOnly` prop to `RadioInput` just cause there was a small error message.

Tested it out and yeah it worked fine for me. Just gotta wait for OMN token deploy.

**Review by pimato — CHANGES_REQUESTED** [review]
Great work. Works without any issues! The only thing I would like to see changed is the dropdown icon styling. I added a playground here to visualize how it should look like:

https://www.figma.com/proto/aTivxuHmp2l6Sn8nho1Ngu/Playground?page-id=2993%3A72064&node-id=2993%3A72080&viewport=-2214%2C424%2C0.29549652338027954&scaling=min-zoom

File: app/src/components/modal/modal_your_connection/index.tsx [hunk 1]
```diff
@@ -85,9 +85,17 @@ const ClaimLeft = styled.div`
...
-const ClaimRight = styled.div`
-  margin-left: auto;
+const ClaimLeftSvg = styled.div`
   display: flex;
+  justify-content: space-between;
+  flex: 1;
+  &:hover {
+    .chevronDown {
+      path {
+        fill: ${props => props.theme.colors.primary};
```
Comment (pimato) [inline app/src/components/modal/modal_your_connection/index.tsx:95]: Hover color is wrong, please use the same hover color as arrow-back and close icon hover color

**Review by pimato — CHANGES_REQUESTED** [review]

**Review by KadenZipfel — APPROVED** [review]
Haven't tested myself but judging by other responses it seems to work great, and the code looks really good 👏

**Pull request merged**